LibreNMS/Alert/Transport/Jira.php

```diff
@@ -1,23 +1,16 @@
 <?php
-/* Copyright (C) 2015 Aldemir Akpinar <aldemir.akpinar@gmail.com>
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <https://www.gnu.org/licenses/>. */
+/*Copyright (c) 2019 GitStoph <https://github.com/GitStoph>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details. */
 
 /**
- * Jira API Transport
+ * Jira Webhook & API Transport
  *
- * @author  Aldemir Akpinar <aldemir.akpinar@gmail.com>
- * @copyright 2017 Aldemir Akpinar, LibreNMS
+ * @author Skylark <https://github.com/LoveSkylark>
```

Comment:
Same here, you can add but not remove.

Reply:
I am not removing any lines or erasing credit from the old transporter, I thew it away and started fresh. 

Code was originally written as a separate JiraWebhooks.php with webhooks only, then I rewrote it to allow for the original connections as well and submitted it as a replacement of for the old Jira transporter. If you look thorough the code only thing that remains are values needed to make it compatible with the old setup (and I only change that after murrant pointed out I would be breaking older Jira deployment).

But I'm happy to change this to anything that gets this approved, but I would suggest @murrant as actual author.